### ISSUE TITLE
typo in external link checker output

### ISSUE THREAD

**Issue by iay** [nanoc/nanoc#1425]
```
    [ ERROR ] external_links - broken reference to http://www.schneier.com/: link have moved permanently to 'https://www.schneier.com/'
```

I think you mean "has moved" (singular).

**Comment by ddfreyne** [comment]
Thanks! I’ve fixed this directly on `master`: 6f2e4c9a59abdabd9f389beb9ac0b21d4ceee458

**Comment by ddfreyne** [comment]
This is now in the Nanoc 4.11.5 release.
